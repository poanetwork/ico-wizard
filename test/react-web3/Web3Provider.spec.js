import React from 'react'
import Adapter from 'enzyme-adapter-react-15'
import { configure, mount, shallow } from 'enzyme'
import Web3Provider from '../../src/react-web3/Web3Provider'
import renderer from 'react-test-renderer'
import DeploymentStore from '../../src/stores/DeploymentStore'

configure({ adapter: new Adapter() })

describe('Web3Provider', () => {
  it(`should render the component `, () => {
    const wrapper = renderer.create(<Web3Provider />)

    expect(wrapper).toMatchSnapshot()
  })
})
