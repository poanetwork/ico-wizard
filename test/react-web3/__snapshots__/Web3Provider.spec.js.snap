// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Web3Provider should render the component  1`] = `null`;
